start.sh

```diff
@@ -5,6 +5,9 @@ if [ ${APP_ROLE:-app} == "sidekiq" ]; then
     bundle exec sidekiq
 else
     echo "starting rails"
+    if [ -f tmp/pids/server.pid ]; then
+      rm tmp/pids/server.pid
```

Comment:
`rm -f tmp/pids/server.pid` will silently fail if the file doesn't exist, then you don't need the `if`